index.toml

```diff
@@ -364,3 +364,13 @@ completion_time = "15 min"
 created_at = 2024-02-12
 haystack_2 = true
 
+[[tutorial]]
+title = "Building Fallbacks to Websearch with Conditional Routing"
+description = "Learn how to direct the query to a web-based RAG route when necessary"
+level = "beginner"
```

Comment:
Would you rather call this 'intermediate'? Up to you, seems like a complex pipeline for beginners